ember-lazy-mount/src/services/engine-loader.js

```diff
@@ -28,7 +27,10 @@ export default class EngineLoaderService extends Service {
     if (this.isLoaded(name)) return;
 
     const owner = getOwner(this);
-    owner.register(`engine:${name}`, require(`${name}/engine`).default);
+    owner.register(
+      `engine:${name}`,
+      globalThis.require(`${name}/engine`).default,
```

Comment:
it is not possible to use `require` under Embroider, so this supports only classic Ember CLI build (as before)